docs/components/modeler/bpmn/undefined-tasks/undefined-tasks.mdx

```diff
@@ -0,0 +1,27 @@
+---
+id: undefined-tasks
+title: "Undefined tasks"
+description: "An undefined task defines a task that is external to the BPM engine."
+---
+
+import myImageUrl from "./assets/undefined-task.png";
+
+An undefined task (also known as abstract task) defines a task for which the type of work is unspecified.
+This is used to model work done by someone the engine does not need to know of and there is no known system or UI interface.
+It is also used when modeling a process that is not automated, or while the process automation is in development.
+
+For the engine, an undefined task is handled as a pass-through activity, automatically continuing the process at the moment the process instance arrives.
+
+<img src={myImageUrl} alt="Example of multiple Undefined Tasks in sequence" />
+
+Undefined tasks have no real benefit for automating processes, and instead provide insights into the tasks that are performed outside of the process engine.
+
+## Additional resources
```

Comment:
Given there is just one additional source currently, I would remove the additional resources header and just leave a header for the XML representation for now.

Reply:
💭 Makes sense to me, but just a heads up that this structure is copied from other BPMN pages. Each has an `## Additional resources` heading, and some only contain the `### XML representation`. Other pages with only that heading: Manual Tasks, Terminate Events, Parallel Gateways, None Events, and Escalation Events

Reply:
Feel free to leave the structure in place, and I'll create an issue to revisit this section for consistency.

Reply:
https://github.com/camunda/developer-experience/issues/51